e2e/provision/init.sh

```diff
@@ -27,6 +27,11 @@ function get_status {
         printf "Memory free(Kb):"
         awk -v low="$(grep low /proc/zoneinfo | awk '{k+=$2}END{print k}')" '{a[$1]=$2}  END{ print a["MemFree:"]+a["Active(file):"]+a["Inactive(file):"]+a["SReclaimable:"]-(12*low);}' /proc/meminfo
     fi
+    if command -v docker >/dev/null; then
+        echo "Docker statistics:"
+        docker stats --no-stream
```

Comment:
@electrocucaracha What is the reason of using it here? 
It means we will have the stats only for a particular moment but not when it keeps on refreshing. Probably it is better to collect every couple of seconds using a cronjob? What do you think?